decoder/completion_resolve.go

```diff
@@ -0,0 +1,20 @@
+package decoder
+
+import (
+	"context"
+	"errors"
+)
+
+// ResolveCandidate tries to gather more information for a candidate,
+// by checking for a resolve hook and executing it.
```

Comment:
```suggestion
// ResolveCandidate gathers more information for a completion candidate
// by checking for a resolve hook and executing it.
// This would be called as part of `completionItem/resolve` LSP method.
```